unit tests/test_process_nextnano_files.py

```diff
@@ -0,0 +1,121 @@
+"""
+Unittests for the module potential.process_nextnano.py.
+
+@author: Zach
+"""
+
+import os, sys
+sys.path.append(os.path.dirname(os.getcwd()))
+
+import unittest
+import numpy as np
+
+import qudipy.potential as pot
+
+'''
+Instructions for running unittests:
+
+1) Open cmd or anaconda promt and change directory to the unittest directory
+2) Make sure tutorial data is downloaded and located in tutorial directory.
+3) Run the unittest via unittest/pytest package:
+    - Unittest is built into python; pytest must be downloaded with 
+    "python -m pip install pytest".
+    - Enter comand: "python -m unittest <filename.py>" or
+        "python -m pytest <filename.py>"
+'''
+
+class ClassTest(unittest.TestCase):
+
+    # Test data is being imported correctly
+    def test_data_importation(self):
```

Comment:
```suggestion
    def test_data_import(self):
```
the word "importation" sounds out of place to me in this context, but it's up to you